spec/built-in-tiles-spec.coffee

```diff
@@ -73,6 +73,14 @@ describe "Built-in Status Bar Tiles", ->
           fileInfo.currentPath.click()
           expect(atom.clipboard.read()).toBe fileInfo.getActiveItem().getPath()
 
+      it "does not leave out the path after #", ->
```

Comment:
Replace the `#` with `a hash` - one of Atom's customizations to Jasmine makes it skip any test that has a `#` in it.